tls/s2n_config.c

```diff
@@ -117,15 +124,15 @@ static int s2n_config_init(struct s2n_config *config)
     config->cert_tiebreak_cb = NULL;
 
     s2n_config_set_cipher_preferences(config, "default");
+    s2n_config_set_signature_preferences(config, "default");
     if (s2n_is_in_fips_mode()) {
         s2n_config_set_cipher_preferences(config, "default_fips");
     }
+
```

Comment:
If I remember correctly, these cases are exclusive? tls13 in fips mode is not something we have to worry about for quite a while?

Reply:
Correct. When we do support that, we'll figure out what its default cipher preference list would look like.